.github/ISSUE_TEMPLATE/feature-request---proposal.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request / proposal
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Is this a breaking change**
+A breaking change would require consumers of the API to modify their code for it to continue to function (ex: renaming of a required field or the change in data type of an existing field). A non-breaking change would allow existing code to continue to function (ex: addition of an optional field or the creation of a new optional endpoint). 
```

Comment:
I think we should add language around implementers as well, per discussion yesterday (cc @hunterowens). E.g. "A a change is breaking if it _requires_ new implementation work from either API consumers or implementers to modify their code in order to continue to consume or produce the spec respectively. Some examples of breaking changes include: removing functionality from the API, major semantic changes to existing functionality, or adding new mandatory functionality that API implementers must provide. Any change that doesn't not require new implementation work (e.g. addition of an optional field or new optional endpoint, or changes to process or documentation) is considered non-breaking."

Reply:
also, this is just an initial stab at this definition, since not everyone was on the call yesterday. I'm going to leave #279 open for further comment if people have concerns with this one